MycroftAI/selene-backend
# Feature/account activity

## Description
Added a new account_activity table to the metrics schema and the logic to populate it.

## How to test
Add/remove accounts/memberships/open dataset agreements and see their effects on the new table.  Perform activity on device that will trigger active counts to increment and see effects on the new table.

## Contributor license agreement signed?
CLA [X] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)
